md/ENVIRONMENT_VARIABLES_SETUP.md

```
@@ -0,0 +1,136 @@
+Kad deployas na production, promijeni:
+
+```env
+NEXTAUTH_URL=https://yourdomain.com
+SMTP_FROM=noreply@yourdomain.com
+```
+
+## ⚠️ SECURITY NOTES
+
+1. **NIKAD** ne commituj `.env.local` u git
+2. Dodaj u `.gitignore`:
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)